Source/Generators/LMCPatchSignalGenerator.hh

```diff
@@ -94,17 +88,16 @@ namespace locust
 
 
             bool DoGenerate( Signal* aSignal );
-            void DriveAntenna(FILE *fp, int PreEventCounter, unsigned index, Signal* aSignal, double* filterarray, unsigned nfilterbins, double dtfilter);
-            void InitializePatchArray();
+            void DriveAntenna(FILE *fp, int PreEventCounter, unsigned index, Signal* aSignal, int nfilterbins, double dtfilter);
+            bool InitializePatchArray();
             bool InitializePowerCombining();
+            FIRHandler fReceiverFIRHandler;
```

Comment:
The FIRHandler changed a little bit where we now have a core class and derived classes, so this needs to change from `FIRHandler fReceiverFIRHandler` to `FIRReceiverHandler fReceiverFIRHandler;`.